app/src/main/java/com/willowtree/vocable/settings/customcategories/adapter/CustomCategoryPhraseAdapter.kt

```diff
@@ -0,0 +1,78 @@
+package com.willowtree.vocable.settings.customcategories.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.isInvisible
+import androidx.recyclerview.widget.RecyclerView
+import com.willowtree.vocable.R
+import com.willowtree.vocable.databinding.EditCustomCategoryPhraseItemBinding
+import com.willowtree.vocable.room.Phrase
+import com.willowtree.vocable.utils.LocalizedResourceUtility
+import org.koin.core.KoinComponent
+import org.koin.core.inject
+
+class CustomCategoryPhraseAdapter(
+    private var phrases: List<Phrase>,
+    private val numRows: Int,
+    private val onPhraseEdit: (Phrase) -> Unit,
+    private val onPhraseDelete: (Phrase) -> Unit
+) : RecyclerView.Adapter<CustomCategoryPhraseAdapter.CustomCategoryPhraseViewHolder>(),
+    KoinComponent {
+
+    private val localizedResourceUtility: LocalizedResourceUtility by inject()
+
+    private var _minHeight: Int? = null
+
+    inner class CustomCategoryPhraseViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private val binding: EditCustomCategoryPhraseItemBinding =
+            EditCustomCategoryPhraseItemBinding.bind(itemView)
+
+        fun bind(phrase: Phrase, onPhraseEdit: (Phrase) -> Unit, onPhraseDelete: (Phrase) -> Unit) {
+            binding.phraseText.text = localizedResourceUtility.getTextFromPhrase(phrase)
+
+            with(binding.actionButtonContainer) {
+                editSayingsButton.action = {
```

Comment:
We might want to rename these buttons to use "phrase" instead of "sayings", for consistency